### ISSUE TITLE
[Accessibility] Specialized question type built on top of a numeric Single-Line Input lacks an accessible name

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#11049]
T26721 - Accessibility Issues with Specialized Question Types https://surveyjs.answerdesk.io/internal/ticket/details/T26721
***
To reproduce the issue, run this demo: [View Plunker](https://plnkr.co/edit/7iHIG2AocvY0Mgtq?preview).
Run the axe Accessibility checking for the page.

A standalone numeric input doesn't have issues, however, a specialized question type built on top of a numeric input doesn't have any title attribute because its `aria-labelledby` attribute references a non-existing element:

<img width="548" height="208" alt="Image" src="https://github.com/user-attachments/assets/e99b4cb7-d83d-4eb3-95ca-8a4d0393dee5" />

<img width="2560" height="1239" alt="Image" src="https://github.com/user-attachments/assets/0b0bbd8e-6d8f-419b-8393-608e59b2321c" />
